app/models/manageiq/providers/amazon/storage_manager/ebs/cloud_volume.rb

```diff
@@ -92,4 +97,17 @@ def provider_object(connection = nil)
     connection ||= ext_management_system.connect
     connection.volume(ems_ref)
   end
+
+  private
+
+  def modify_volume_options(options = {})
+    modify_opts = {}
+    if volume_type != 'standard'
+      modify_opts[:volume_type] = options[:volume_type] if options[:volume_type] && options[:volume_type] != 'standard'
+      modify_opts[:size]        = options[:size] if options[:size] && options[:size] != (size / 1_073_741_824)
```

Comment:
true, alternative check would be `... && options[:size].gigabyte == size`. Database stores the size in bytes, but the `:size` in options must be in gigabytes.